compiler/fir/checkers/checkers-component-generator/src/org/jetbrains/kotlin/fir/checkers/generator/diagnostics/FirDiagnosticsList.kt

```diff
@@ -68,6 +69,18 @@ object DIAGNOSTICS_LIST : DiagnosticList() {
         val FLOAT_LITERAL_OUT_OF_RANGE by error<PsiElement>()
         val WRONG_LONG_SUFFIX by error<KtElement>(PositioningStrategy.LONG_LITERAL_SUFFIX)
         val DIVISION_BY_ZERO by warning<KtExpression>()
+        val VAL_OR_VAR_ON_LOOP_PARAMETER by warning<KtParameter>(PositioningStrategy.VAL_OR_VAR_NODE) {
```

Comment:
Should this be an error instead of warning? Same below.

Reply:
According to the diagnostics list doc, we can downgrade to a warning. I believe it is @mglukhikh who put that note.